plugin-api/src/main/java/org/hyperledger/besu/plugin/data/TransactionSelectionResult.java

```diff
@@ -53,9 +53,10 @@ protected interface Status {
   private enum BaseStatus implements Status {
     SELECTED,
     BLOCK_FULL(true, false),
+    BLOBS_FULL(false, false),
     BLOCK_OCCUPANCY_ABOVE_THRESHOLD(true, false),
     BLOCK_SELECTION_TIMEOUT(true, false),
-    TX_EVALUATION_TOO_LONG(false, true),
```

Comment:
we can safely stop selected for this selection results, since it means that the tx took more that the block selection time out to evaluate, and so does not make sense to continue the evaluation